src/test/java/com/checkmarx/flow/cucumber/integration/cxconfig/CxConfigSteps.java

```diff
@@ -71,19 +72,22 @@
     private ResultsService resultsService;
     private Boolean isPullRequestApproved;
 
-    private FlowService flowService;
+    private final FlowService flowService;
     private String branch;
     private ScanRequest request;
+    private final JiraProperties jiraProperties;
 
     public CxConfigSteps( FlowProperties flowProperties, GitHubService gitHubService,
```

Comment:
Question - why init all properties via the constructor and not by using the @Autowired annotation?
Is Spring initiate it?

Reply:
This is actually Orly's code, but I also prefer constructor injection.
Even the Spring team itself "generally advocates constructor injection" (https://docs.spring.io/spring-framework/docs/current/spring-framework-reference/core.html#beans-setter-injection)